src/i18n/pl_PL.ts

```diff
@@ -0,0 +1,976 @@
+        closeWarning: {
+            title: 'Niezapisane zmiany',
+            text:
+                'Wprowadziłeś zmiany w App Store, które nie zostały jeszcze zapisane. Zresetuj je lub zapisz, aby zamknąć App Store.',
+            close: 'Zamknij',
+        },
+    },
+    settings: {
+        name: 'Ustawienia',
+        save: 'Zapisz',
+        discard: 'Anuluj',
+        reset: 'Reset',
+        export: 'Export',
+        import: 'Import',
+        resetWarning: {
+            title: 'Resetuj ustawienia',
+            text:
+                'Czy na pewno chcesz zresetować ustawienia do wartości domyślnych? Tego nie można cofnąć!',
+            close: 'Anuluj',
+            total: 'Wszystkie ustawienia',
+            module: 'Tylko dla tego modułu',
+        },
+        resetWarningSetting: {
+            title: 'Resetuj ustawienia',
+            text:
+                'Czy na pewno chcesz zresetować to ustawienie<b>{setting}</b>modułu<b>{module}</b>do jego wartości domyślnej?',
+            close: 'Anuluj',
+            reset: 'Reset',
+        },
+        closeWarning: {
+            title: 'Niezapisane zmiany',
+            text:
+                'Dokonałeś zmian w ustawieniach, które nie zostały jeszcze zapisane. Zresetuj je, odrzuć lub zapisz, aby zamknąć ustawienia.',
+            close: 'Zamknij',
+        },
+        changeList: {
+            true: 'On',
+            false: 'Off',
+        },
+    },
+} as { [moduleId: string]: { [key: string]: unknown } };
+moduleRootFiles
+    .keys()
+    .forEach(key => (modules[key.split('/')[2]] = moduleRootFiles(key)));
+
+const t = {} as { [key: string]: unknown };
+
+// Commented as dir ./en_US does not exist currently
+// furtherFiles
+//     .keys()
+//     .forEach(
+//         key => (t[key.split('/')[1].replace(/\..*$/, '')] = furtherFiles(key))
+//     );
+
+export default {
+    modules,
+    ...t,
+    error: {
+        title: 'LSS Manager: Błąd',
+        msg:
+            'Jeśli ten błąd występuje często, zgłoś go zespołowi LSSM!',
+    },
+    warnings: {
+        version: {
+            title: 'Zła wersja LSS Manager',
+            text:
+                'Drogi użytkowniku, niestety odkryliśmy, że nie masz najnowszej wersji LSS Manager. Najnowsza wersja to {version}, ale ty masz {curver}. Proszę przeładować grę  (Ctrl + F5, na urządzeniach Apple polecenie + R), to powinno naprawić błąd. Jeśli błąd będzie się powtarzał, zgłoś go zespołowi! W przypadku użycia złej wersji nie możemy zagwarantować pełnej funkcjonalności LSS-Manager.',
+            close: 'Zamknij wiadomość i przeładuj grę (zalecane)',
+            abort: 'Zmaknij wiadomość bez przeładowania',
+        },
+    },
+    globalSettings: {
+        name: 'Ustawienia główne',
+        labelInMenu: {
+            title: 'Etykieta zamiast ikony w menu',
+            description:
+                'Wyświetla prostą etykietę na pasku nawigacji zamiast logo LSSM',
+        },
+    },
+    vehicles: {

```

Comment:
please check this on the vehicle ID's and names.
https://github.com/LSS-Manager/lss-manager-v3/blob/6d3cae5fa634c1251470c3f23d0f3430fbc1f40c/lss-manager-v3/helperfunctions.js#L246

Reply:
Also for the buildings

Reply:
ok

Reply:
The vehicle list is not correct yet. Please be sure you use the correct order, ID's and names and other needed information is correct. Vehicles with schooling needs `schooling` thats the same as what's shown ingame when a schooling is started. shownSchooling needs to be the same as the schooling in your personal list at the building.